README.md

```diff
@@ -11,7 +11,7 @@
 This is a program used to manage your qBittorrent instance such as:
 
 * Tag torrents based on tracker URLs
-* Update categories based on save directory
```

Comment:
Maybe we shouldn't remove this line since the category section does update the torrents category based on the save directory (cat-update) which is a different function than the cat_change

Reply:
Ah yeah noticed that later.

Will take a stab at this later this week